src/ert/scheduler/slurm_driver.py

```diff
@@ -154,8 +155,10 @@ def _submit_cmd(
             sbatch_with_args.append(f"--nodelist={self._include_hosts}")
         if self._exclude_hosts:
             sbatch_with_args.append(f"--exclude={self._exclude_hosts}")
-        if self._max_runtime:
-            sbatch_with_args.append(f"--time={self._max_runtime}")
+        if self._max_runtime and int(self._max_runtime):
```

Comment:
Hm, this just converts it to int? so it gets false only when _max_runtime==0 right?

Reply:
This makes it skip the `--time=` argument when the timeout is less than 1 seconds. If not it will add `--time=0:00:00` to `sbatch` which I think makes `sbatch` error.

Reply:
Since the incoming _max_runtime is a float, this must be checked.

Reply:
I see. :+1: